std_msgs/Header header
uint32 tracking_accuracy

#tracking_accuracy types constants
int32 TRACKING_ACCURACY_FAILED=0
int32 TRACKING_ACCURACY_LOW=1
int32 TRACKING_ACCURACY_MEDIUM=2
int32 TRACKING_ACCURACY_HIGH=3
